WordPress/src/main/res/values/strings.xml

```diff
@@ -1265,7 +1265,9 @@
     <string name="stats_view_search_terms">Search Terms</string>
     <string name="stats_view_publicize">Publicize</string>
     <string name="stats_view_followers">Followers</string>
-    <string name="stats_view_follower_totals">Follower Totals</string>
```

Comment:
You don't have to rename the existing card I guess.  The new name is only for new card!

Reply:
Done! 686be299d1f9e29189eb64458f7483f74fd588a6

Reply:
I didn't think it as a new card, so I replaced the old card. And I didn't also add a new `InsightType` to `StatsStore`. But maybe it's better to add `TOTAL_FOLLOWERS` to [InsightType](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/c61b3e9486d4030bd08de89c58badcc085a089d6/fluxc/src/main/java/org/wordpress/android/fluxc/store/StatsStore.kt#L172) in `StatsStore`. Wdyt?

Reply:
Added new `InsightType` to FluxC for total followers card. https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2377
I updated the code to use the new InsightType `TOTAL_FOLLOWERS`, instead of `FOLLOWER_TOTALS` if the flag is enabled. 3177a0dc069f7c60401a03db5f51425020eeab8c